src/components/@shared/FormInput/InputElement/Radio/index.tsx

```diff
@@ -32,7 +34,7 @@ export default function InputRadio({
               })}
               htmlFor={slugify(option)}
             >
-              {option}
+              {option} {postfix}
```

Comment:
suggestion: update in JSON would enable us to link in the Radio / Checkbox components respectively.

e.g.
```jsx
<label>
  {prefix[index]}{' '}
  {action[index] ? 
    <Link to={action[index]}>
      {option}
    </Link>
  :
    {option}
  }{' '}
  {postfix[index]}
</label>
```

It could make sense to move this to it's own option component
```jsx
<Option value={option} prefix={prefix[index]} postfix={postfix[index]} action={action[index]} />
```